### ISSUE TITLE
Whitelist DailyDivs.com

### ISSUE THREAD

**Issue by dailydivs** [MetaMask/eth-phishing-detect#2412]
Hi, we would like to have all of our domains Whitelisted in Metamask. There is no malicious code in our files that phish for users information or puts their ETH at risk of being taken. According to members from our community, there is an attack on our reputation by users outside of our community who are, in a lack of better words, jealous and envious of our growth and success.

You can see the amount of usage on DappRadar here: https://dappradar.com/app/877/dailydivs
The above clearly shows the amount of transactions and engagement on our contracts. With the exception of users not using enough gas/gwei to process transactions, there are no issues present.

The following domains need to be whitelisted:
https://dailydivs.com
https://earn.dailydivs.com
https://lotto.dailydivs.com
https://lending.dailydivs.com
https://exchange.dailydivs.com

**Comment by fomoprincess** [comment]
I have nothing but great things to say about my experience with Daily Divs. The develop has created and awesome platform and I happy to be a part of this community.  Please expedite this!

**Comment by Kail68** [comment]
I've been since the start with absolutely zero issues. Contracts work as advertised.

**Comment by LIlCryptoMitch** [comment]
Zero issues with DailyDivs,

**Comment by HalfManHalfCrypt0** [comment]
daily divs is doing great, easy, honest, and no problems at all. please whitelist all daily divs domains.

**Comment by xcrypto247** [comment]
DailyDivs has outstanding services & their community is great!

**Comment by cryptodollar1** [comment]
I have ZERO issues with DailyDivs.com! This website is amazing in all facets. Daily Divs is amazing!

**Comment by DDTHeem** [comment]
Daily Divs has been an amazing platform with 0 issues for me thus far. I believe this block request to be an obvious ploy by the competition at P3D and other similar dapps that feel threatened and will like to stop DDT's progress.

**Comment by afrketich** [comment]
Daily Divs has been great, and the dev is very open and communicative.  I don't see any reason why it should be blocked, and anyone claiming it should be is just jealous of the legitimate success of the platform.  It is easy to use, the community and backers are supportive and helpful, and I am fully confident that my funds are safe.

**Comment by Bebocas** [comment]
Dear Sirs,
My experience with Daily Divs platform has been a brilliant one with zero issues . The developer has done a superb  job and created and awesome flawless platform with a great support and  community which I am happy to be  a part of. Please expedite this!

**Comment by TheDazzle101** [comment]
There must be some haters out there trying to slow DailyDivs down, but there are absolutely no problems with it, and I'm blown away how fast our community is growing with more interest coming all the time, please stop the hiccups and do not let these people spread false info about DailyDivs. Thank you very much!

**Comment by HasteDrei** [comment]
Daily Divs platform seems fine to me, no problems. The developer has put together a great site and contract delivers as promised. Please, WhiteList site so our community can continue to grow. Thank you.

**Comment by TheDazzle101** [comment]
Yes please whitelist all connections to site, it is running perfect, and
anyone saying otherwise obviously has another agenda.

**Comment by Munters13** [comment]
totally agree with all above. Daily Divs has been nothing short of an amazing platform.Those people saying bad things are only trying to derail a great platform.

**Comment by nolemartley** [comment]
I've had no issues with this domain and its hosted content. Please do not blacklist dailydivs.

**Comment by b0nfiree** [comment]
![fp3891](https://user-images.githubusercontent.com/1980573/45520820-22613a80-b789-11e8-96e7-4425ec3ca747.png)

**JAYSON IS A KNOWN FRAUDSTER

THERE IS A REASON THIS WAS FLAGGED AND BLOCKED BY METAMASK

WITHDRAW YOUR ETH BEFORE IT IS TOO LATE.**

0xdogecoin.com    2018-06-04
0xdollar.com    2018-06-10
blockclout.com    2018-09-10
coinkarma.me    2018-09-06
coinreward.me    2018-09-06
dailydivs.com    2018-09-12
etcfomo.com    2018-08-20
etherdank.com    2018-09-10
gpumining.xyz    2018-05-26
hbcucoins.com    2018-09-10
theilluminati.io
0xbitcoincash
proof of treveon james
fomoloop 
h4d

All of these sites were registered by the creator of dailydivs. He has run off with hundreds of ETH already and will gladly do it again.

THE CONTRACTS AREN'T EVEN VERIFIED.

THE SMART LENDING IS A SCAM.

JUST LIKE BITCONNECT.

READ THIS:

Earn 1.33% everyday through the technology of the A.I. of our smart contract. Smart lending is designed to earn income for DDT token holders forever! No humans can stop this contract or your earnings! Using Smart Lending is simple, just buy DDT Tokens on the exchange and deposit them here in Smart Lending to start earning 39.9% per month with no mysterious trading bot needed. Your profit can be withdrawn at anytime with no fee and your profit is accrued by the second in your profit wallet!

**GUARANTEED RATES OF RETURN ARE SCAMS!!!**

Spydr / Jayson / Banksy they're all the same person. Don't get fooled!!!

**Comment by enkigod** [comment]
Please white list DailyDivs.com daily divs, the site has been up with no problems and the site is legit. All of the hosted content is legit and great.  I have over 30 years experience in the IT field and have read the code and its 100% safe.  The community is great and the platform is helping so many people.  This is an outside attack from jealous people.

**Comment by blackcrypto8** [comment]
This one of the best plateforms with a strong community behind it. please whitelist.

**Comment by cryptorenzzz** [comment]
I’ve been in this since the first two hours, I’ve had no problems I’ve been able to withdraw reinvest and deposit without any issues or virus alerts. People outside the community have been trying to sabotage DailyDivs because how far it has come in such a short time, but that’s because the devs and community are very supportive of this in succeeding so ignore the lies and fake picture claims of grown adults acting like little toddlers throwing tantrums -.-“

**Comment by CryptoChronologist** [comment]
We still have one piece of DailyDivs in need of whitelisting. Would you be so kind as to get this done too?

https://earn.dailydivs.com

**Comment by dbzcri** [comment]
Daily divs is an okay experience . Good platform awesome community

**Comment by Boozer79** [comment]
Hey, I've been in Daily Divs from the first day it opened and this DAP hasn't missed a beat. I have not had a single issue with either the platform or the developer. The games are great and the community that we have building around us is fantastic. The 1 and only negative comment here is the exact same comment made on a YouTube video about Daily Divs, it's obviously from someone working on a DAP similar to this one, trying to bring us down.

**Comment by noFUD** [comment]
no problems with this platform great project

**Comment by jaysonfraudster** [comment]
![image](https://user-images.githubusercontent.com/43264892/45537451-d967bf80-b836-11e8-804d-b391f56634d4.png)

Jayson is a fraudster.

He has exitscammed on alot of people before.

0xdogecoin.com 2018-06-04
0xdollar.com 2018-06-10
blockclout.com 2018-09-10
coinkarma.me 2018-09-06
coinreward.me 2018-09-06
dailydivs.com 2018-09-12
etcfomo.com 2018-08-20
etherdank.com 2018-09-10
gpumining.xyz 2018-05-26
hbcucoins.com 2018-09-10
theilluminati.io
0xbitcoincash
proof of treveon james
fomoloop
h4d

Kail68 - paid shillers / marketers
xcrypto247 - paid shillers / marketers

**Comment by niko011** [comment]
Very good experience with Daily Divs, no issues at all.

**Comment by jaysonfraudster** [comment]
Majority of the people posting here are all his paid promoters from youtube and discord.

Please log down all their IP addresses.

This is crucial evidence when time comes.

**Comment by Boozer79** [comment]
> Majority of the people posting here are all his paid promoters from youtube and discord.
> 
> Please log down all their IP addresses.
> 
> This is crucial evidence when time comes.

No payment being received here. I'm just a happy investor that happened to get in at the right time this time, don't want to see haters destroy a good thing for myself and the community.

**Comment by jaysonfraudster** [comment]
He created all the existing scam and your supporting this ?

**Comment by dAppBridge** [comment]
Surely the contract should be verified any open - way too risky to have that amount of $ETH in a closed source contract.  There could be any number of exit scam methods within it.

**Comment by Boozer79** [comment]
Yeah, I agree but there is a good reason. Keeping it private for now means all the copycat DAPS can’t muscle in on Daily Divs success. I’m sure in the future it will be verified.

**Comment by dAppBridge** [comment]
What about what @jaysonfraudster says (https://github.com/MetaMask/eth-phishing-detect/issues/2412#issuecomment-421264898)

https://viewdns.info/reverseip/?host=dailydivs.com&t=1

**Comment by Boozer79** [comment]
Really, I don't know. All I do now is that Jayson has already explains this in one of his YouTube interview videos.

**Comment by cryptomaniac1977** [comment]
I am a very sceptical person and find it hard to trust platforms. Before i invested in this program i had a friend of mine check the smart contract of the exchange game and he advised that it is solid code. Now since i have been in daily divs i have had absolutely no problems whatsoever and the community is great, the developer, Jayson, always listens and helps people out and i cant say anything negative about the whole set up here. With the growing success of daily divs in the back of my mind i did start to think that there are going to be people from the other competing platforms causing a lot of fud and trying to derail this platform. I anticipated that and it is normal for these kind of things to happen but i am here to say that this platform has been great and never had one problem. All my questions and concerns were taken care of, so white listing Dailydivs is the right thing to do. Thanks

**Comment by CryptoChronologist** [comment]
> ![image](https://user-images.githubusercontent.com/43264892/45537451-d967bf80-b836-11e8-804d-b391f56634d4.png)
> 
> **Jayson is a fraudster. He created all the existing scams.**
> 
> **Proof is above. This guy is going to jail just like the creators of BitConnect.**
> 
> **Someone please notify the Feds !**
> 
> **This guy has stolen alot of money from alot of people !**
> 
> He has exitscammed on alot of people before.
> 
> 0xdogecoin.com 2018-06-04
> 0xdollar.com 2018-06-10
> blockclout.com 2018-09-10
> coinkarma.me 2018-09-06
> coinreward.me 2018-09-06
> dailydivs.com 2018-09-12
> etcfomo.com 2018-08-20
> etherdank.com 2018-09-10
> gpumining.xyz 2018-05-26
> hbcucoins.com 2018-09-10
> theilluminati.io
> 0xbitcoincash
> proof of treveon james
> fomoloop
> h4d
> 
> **Kail68 - paid shillers / marketers xcrypto247 - paid shillers / marketers TheDazzle101 - paid shillers / marketers**
> 
> Admin, please check their discord channel.
> 
> Majority of the users posting here are paid promoters from his channel.
> 
> His know to hire youtubers / discord shillers to shill his platform.
> 
> **Spydr / Jayson / Banksy they're all the same person.**

Wait, this is the guy from Zethr, he just changed his screenname...LOL
None of us are paid here dude

**Comment by sirpixalok** [comment]
I have no problem with Daily Divs. It is no different than P3D and that was not blacklisted. I have not been scammed and have actually broken even plus some from what I have put in. Everything in crypto is risky, but just because it’s risky doesn’t mean it needs blacklisted. Please whitelist the requested domains. We are all adults here. If I want to throw my ether into a platform I should be able to do so. That’s my decision to make.

**Comment by dailydivs** [comment]
@409H Please review and resolve.

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/2427
